plugins/hypervisors/kvm/src/com/cloud/hypervisor/kvm/resource/wrapper/LibvirtPvlanSetupCommandWrapper.java

```diff
@@ -87,7 +88,7 @@ public Answer execute(final PvlanSetupCommand command, final LibvirtComputingRes
             } else if (command.getType() == PvlanSetupCommand.Type.VM) {
                 final String ovsPvlanVmPath = libvirtComputingResource.getOvsPvlanVmPath();
 
-                final Script script = new Script(ovsPvlanVmPath, timeout, s_logger);
+                final Script script = new Script(ovsPvlanVmPath, timeout.getMillis(), s_logger);
```

Comment:
Please consider adding an overridden version of the `Script(String, int, Logger)` constructor that accepts `Duration` to encapsulate this type conversion.